Presentation.Web/Controllers/API/V1/OrganizationController.cs

```diff
@@ -70,6 +70,33 @@ public HttpResponseMessage GetBySearch(string q, int orgId, int take = 25)
             }
         }
 
+        public override HttpResponseMessage GetSingle(int id)
+        {
+            try
+            {
+                var item = Repository.GetByKey(id);
+
+                if (item == null)
+                {
+                    return NotFound();
+                }
+
+                if (!AllowRead(item))
+                {
+                    return Forbidden();
+                }
+
+                var dto = Map(item);
+                dto.CanCvrBeModified = _organizationService.CanActiveUserModifyCvr();
```

Comment:
I think it would be better to expose this info on a different endpoint. Right now you are duplicating a lot of logic just to insert a property which does not belong on the RESTful resource